### PR TITLE
added forms

### PR DESCRIPTION
**Pull request by Ankur9669**
Please scroll to the last in the documentation page to find forms

### CODE REVIEW

File: documentation.html [hunk 1]
```diff
@@ -1114,11 +1120,554 @@ <h4 class="component-heading">Lists</h4>
...
+                  href="forgotpassword.html"
+                >
+                  Forgot Password
+                </a>
+              </div>
+              <button class="button button-primary form-cta-button">
+                Login
+              </button>
+              <a class="button button-outline-primary form-cta-button" href="#">
+                SignUp With Us
```
Comment (Shivani-1524) [inline documentation.html]: Just a suggestion to be written as Sign Up instead.

File: documentation.html [hunk 2]
```diff
@@ -1114,11 +1120,554 @@ <h4 class="component-heading">Lists</h4>
...
...2520%2520%2520%253Cspan%253E%250A%2520%2520%2520%2520%2520%2520%253Cinput%2520type%253D%2522checkbox%2522%2520value%253D%2522Remember%2520Box%2522%2520%252F%253E%250A%2520%2520%2520%2520%2520%2520%253Clabel%2520for%253D%2522Remember%2520Box%2522%2520class%253D%2522font-medium%2522%250A%2520%2520%2520%2520%2520%2520%2520%2520%253ERemember%2520Me%253C%252Flabel%250A%2520%2520%2520%2520%2520%2520%253E%250A%2520%2520%2520%2520%253C%252Fspan%253E%250A%2520%2520%2520%2520%253Ca%250A%2520%2520%2520%2520%2520%2520class%253D%2522forgot-password-btn%2520font-medium%2522%250A%2520%2520%2520%2520%2520%2520href%253D%2522forgotpassword.html%2522%250A%2520%2520%2520%2520%253E%250A%2520%2520%2520%2520%2520%2520Forgot%2520Password%250A%2520%2520%2520%2520%253C%252Fa%253E%250A%2520%2520%253C%252Fdiv%253E%250A%2520%2520%253Cbutton%2520class%253D%2522button%2520button-primary%2520form-cta-button%2522%253E%250A%2520%2520%2520%2520Login%250A%2520%2520%253C%252Fbutton%253E%250A%2520%2520%253Ca%2520class%253D%2522button%2520button-outline-primary%2520form-cta-button%2522%2520href%253D%2522%2523%2522%253E%250A%2520%2520%2520%2520SignUp%2520With%2520Us%250A%2520%2520%253C%252Fa%253E%250A%253C%252Fform%253E"
```
Comment (Shivani-1524) [inline documentation.html]: missed lazy loading here

File: documentation.html [hunk 3]
```diff
@@ -1114,11 +1120,554 @@ <h4 class="component-heading">Lists</h4>
...
+            <form class="authentication-form">
+              <h2
+                class="font-large weight-semi-bold authentication-form-heading"
+              >
+                <span class="logo-text font-large">FiT</span>BiT
+              </h2>
+              <p
+                class="font-medium-large weight-semi-bold authentication-form-sub-heading"
+              >
+                LOGIN INTO YOUR ACCOUNT
```
Comment (Shivani-1524) [inline documentation.html]: just  a suggestion but Login is more preferred these days as it is concise and delivers the same meaning to the user

File: Components/Forms/forms.css [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+    border-radius: 6px;
+    padding: 1rem 0.5rem;
+    box-shadow: rgba(0, 0, 0, 0.16) 0px 3px 6px, rgba(0, 0, 0, 0.23) 0px 3px 6px;
+}
+.authentication-form-heading{
+    text-align: center;
+    margin: 0rem 1rem 0.3rem 1rem;
+}
+.authentication-form-sub-heading{
+    margin: 0.3rem 1rem;
```
Comment (gauravsinhaweb) [inline Components/Forms/forms.css]: use short classname for short typing. authentication => auth

File: Components/Modal/modal.css [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+    width: 100%;
+    height: 100%;
+    background-color: rgb(0, 0, 0); /* Fallback color */
+    background-color: rgba(0, 0, 0, 0.4); /* Black w/ opacity */
+    z-index: 100;
+}
+.modal-content{
+    width: 500px;
+    background-color: var(--background-color);
+    min-height: 250px;
```
Comment (gauravsinhaweb) [inline Components/Modal/modal.css]: use rems  for responsiveness

**Review by gauravsinhaweb — CHANGES_REQUESTED** [review]

File: documentation.html [hunk 6]
```diff
@@ -1114,11 +1120,554 @@ <h4 class="component-heading">Lists</h4>
...
+                  class="font-medium"
+                />
+                <i class="fa fa-envelope-o" aria-hidden="true"></i>
+              </div>
+              <div class="form-input-box">
+                <input
+                  type="password"
+                  placeholder="Password"
+                  class="font-medium"
+                />
```
Comment (Naman-Saxena1) [inline documentation.html]: You can add labels to input fields, as Danish suggested us.

**Review by Naman-Saxena1 — CHANGES_REQUESTED** [review]

File: documentation.html (hunk 6 above)
Comment (Ankur9669) [reply documentation.html]: I did'nt understand it.

File: documentation.html (hunk 1 above)
Comment (Ankur9669) [reply documentation.html]: Done

File: documentation.html (hunk 3 above)
Comment (Ankur9669) [reply documentation.html]: Done

File: documentation.html (hunk 2 above)
Comment (Ankur9669) [reply documentation.html]: Done

File: Components/Forms/forms.css (hunk 4 above)
Comment (Ankur9669) [reply Components/Forms/forms.css]: Done

File: Components/Modal/modal.css (hunk 5 above)
Comment (Ankur9669) [reply Components/Modal/modal.css]: Done

File: documentation.html (hunk 6 above)
Comment (Naman-Saxena1) [reply documentation.html]: I was talking about input labels are good practices for accessibility.
https://www.w3schools.com/tags/tag_label.asp

Comment (Ankur9669) [reply documentation.html]: Ya but here we don't need labels
Placeholder would only work

Comment (Ankur9669) [reply documentation.html]: But i will add a name attribute

**Pull request merged**